snuba/api.py

```diff
@@ -318,7 +313,7 @@ def parse_and_run_query(dataset, validated_body, timer):
         'referrer': request.referrer,
         'num_days': (to_date - from_date).days,
         'num_projects': len(project_ids),
-        'sample': sample,
+        'sample': body.get('sample'),
     }
```

Comment:
This is really just a grab bag of random attributes at the moment, we should figure out a more reasonable thing to do here.